data/human/free worlds start.txt

```diff
@@ -1075,6 +1081,14 @@ mission "FW Senate 1B"
 			`	"Indeed," he says. "Anyway, I'm off to a meeting, but meet me later in the spaceport if you've got room for some passengers."`
 
 
+mission "FW Senate Timer"
```

Comment:
Instead of a separate mission, my plan was to add `event "fw senate timer" 8` (8 days because you're technically not late if it takes you exactly 7 days) to the `on offer` of the mission. Then you can branch based off of `has "event: fw senate timer"`. Would only be a line to define the event and a line to set the event as opposed to the eight lines you have for this mission.